src/main/scala/nl.knaw.dans.easy.authinfo/components/FileItem.scala

```diff
@@ -59,12 +59,11 @@ object FileItem {
   private def solr2jsonKey(key: String): String = solr2JsonKeys.getOrElse(key, key)
 
   def toJson(solrDocument: SolrDocument): JsonAST.JObject = {
-    val fieldValueMap = solrDocument.getFieldValueMap
-    fieldValueMap
+    solrDocument.getFieldValueMap
       .keySet()
       .asScala // asScala on the map throws UnsupportedOperationException
       .filter(_.matches("(id|easy_.*)")) // filter on the solr query would spread knowledge
-      .map(key => (solr2jsonKey(key), fieldValueMap.get(key).toString))
+      .map(key => (solr2jsonKey(key), solrDocument.getFieldValueMap.get(key).toString))
```

Comment:
This might help a little for those items that don't match the filter. But did you try to find the main culprit with https://alvinalexander.com/scala/how-show-memory-ram-use-scala-application-used-free-total-max/